test/fixtures/forms/password.html

```diff
@@ -0,0 +1,5 @@
+<label class="textfield textfield--label_bordered">
```

Comment:
Is it possible to have this template outside the test? If we change layout than test will still pass, because we have fixture in test.

Reply:
Won't do for now, I'm over time for this and need to work on other things.